jbehave-core/src/main/java/org/jbehave/core/model/ExamplesTable.java

```diff
@@ -506,6 +530,23 @@ public Optional<String> getNullPlaceholder() {
             return Optional.ofNullable(properties.getProperty(NULL_PLACEHOLDER_KEY));
         }
 
+        public boolean isProcessEscapeSequences() {
+            String processEscapeSequences = properties.getProperty(PROCESS_ESCAPE_SEQUENCES_KEY);
+            if (processEscapeSequences != null) {
+                String processEscapeSequencesTrimmed = processEscapeSequences.trim();
+                if ("true".equalsIgnoreCase(processEscapeSequencesTrimmed)) {
```

Comment:
by the code it looks more efficient if we invoke toLowerCase() on processEscapeSequences.trim() result and just invoke here "true".equals(processEscapeSequencesTrimmed)